### DISCUSSION TITLE
Where can I find a list of Local with the full country code(Bonus for including the timezone)?

### DISCUSSION THREAD

**Discussion by Oceanestars** [js-fns/date-fns#3282] [Q&A] [answered]
Hi, I'm not the most familiar with timezone and country code. 
I have found the list of locale here: https://github.com/date-fns/date-fns/blob/b8b7e521247f65d464712f5499de14e97bbef3bf/src/locale/index.ts or even looking at the folders https://github.com/date-fns/date-fns/tree/b8b7e521247f65d464712f5499de14e97bbef3bf/src/locale but I don't know what most of these stand for. I can figure out what CY stand for when I click in the folder and look at the top of `snpachot.md`.  Is there an easier list available that says something  like:
CY -> Welsh (cy) (Timezone: EET)

**Comment by fturmel** [comment, marked as answer]
I don't know why it didn't get accepted, but PR #2838 was meant to add a dedicated page in the docs for this.

It's a bit outdated now, but you can preview the Markdown output of the PR here: https://github.com/date-fns/date-fns/blob/af38b9eb940cdceae62e8742fd67161b47740bde/docs/i18nLocales.md

Time zones and locales are distinct things.

`date-fns` uses the Unicode CLDR as a reference for locales
https://en.wikipedia.org/wiki/Common_Locale_Data_Repository

And time zones in the JavaScript runtimes and `Intl` are based on the IANA time zone database
https://www.iana.org/time-zones
https://en.wikipedia.org/wiki/List_of_tz_database_time_zones

Hope that helps!

**Reply by Oceanestars** [reply]
Thank you so much! That was super helpful and an awesome answer, truly appreciate it!
